docs/operations.md

```diff
@@ -123,6 +171,128 @@ stays in the database:
 
 **Default**: `30` (days)
 
+### Scavenge database option
+
+#### Always keep scavenged (_Deprecation Notice_)
+
+This option ensures that the newer chunk from a scavenge operation is always kept.
+
+| Format               | Syntax                             |
+|:---------------------|:-----------------------------------|
+| Command line         | `--always-keep-scavenged`          |
+| YAML                 | `AlwaysKeepScavenged`              |
+| Environment variable | `EVENTSTORE_ALWAYS_KEEP_SCAVENGED` |
+
+#### Scavenge backend page size
+
+Specify the page size of the scavenge database.The default value is 16KiB.
+
+| Format               | Syntax                                  |
+|:---------------------|:----------------------------------------|
+| Command line         | `--scavenge-backend-page-size`          |
+| YAML                 | `ScavengeBackendPageSize`               |
+| Environment variable | `EVENTSTORE_SCAVENGE_BACKEND_PAGE_SIZE` | 
+
+#### Scavenge backend cache size
+
+Specify the amount of memory, in bytes, to use for backend caching during scavenging. The default value is 64MiB.
+
+| Format               | Syntax                                   |
+|:---------------------|:-----------------------------------------|
+| Command line         | `--scavenge-backend-cache-size`          |
+| YAML                 | `ScavengeBackendCacheSize`               |
+| Environment variable | `EVENTSTORE_SCAVENGE_BACKEND_CACHE_SIZE` |                       
+
+#### Scavenge hash users cache capacity
+
+Specify the number of stream hashes to remember when checking for collisions.
+
+| Format               | Syntax                                          |
+|:---------------------|:------------------------------------------------|
+| Command line         | `--scavenge-hash-users-cache-capacity`          |
+| YAML                 | `ScavengeHashUsersCacheCapacity`                |
+| Environment variable | `EVENTSTORE_SCAVENGE_HASH_USERS_CACHE_CAPACITY` |
+
+## HTTP option
```

Comment:
some of the things in this section are options but some aren't. it would probably be nice to just keep this to options that you can provide when starting a scavenge (because thats the only one with lots of options) and explain the other bits elsewhere.

for example the 'Stop current scavenge' section below isn't an option, but there is a section above on stopping a scavenge that we could add the information to instead